lib/components/logIn.dart

```diff
@@ -35,232 +39,287 @@ class _LogInScreenState extends State<LogInScreen> {
-                            width: MediaQuery.of(context).size.width,
-                            height: MediaQuery.of(context).size.height,
-                            decoration: BoxDecoration(
-                              color: Color(0xff8B4513),
+          backgroundColor: Colors.black38,
+          strokeWidth: 5.0,
+        ),
+        child: Scaffold(
+          body: SingleChildScrollView(
+            child: Container(
+              width: MediaQuery.of(context).size.width,
+              height: MediaQuery.of(context).size.height,
+              child: Form(
+                key: _logInKey,
+                child: Column(
+                  children: <Widget>[
+                    Container(
+                      height: MediaQuery.of(context).size.height * 0.4,
+                      child: Stack(
+                        children: <Widget>[
+                          Positioned.fill(
+                            child: ClipPath(
+                              clipper: oclip(),
+                              child: Container(
+                                width: MediaQuery.of(context).size.width,
+                                height: MediaQuery.of(context).size.height,
+                                decoration: BoxDecoration(
+                                  color: Color(0xff8B4513),
+                                ),
+                              ),
                             ),
                           ),
-                        ),
-                      ),
-                      Positioned.fill(
-                        child: ClipShadow(
-                          boxShadow: [
-                            BoxShadow(
-                              offset: Offset(-10.0, -200.0),
-                              blurRadius: 0.0,
-                              spreadRadius: 0.0,
-                              color: Color(0x4f1e4d2b),
-                            )
-                          ],
-                          clipper: rclip(),
-                          child: Container(
-                            width: MediaQuery.of(context).size.width,
-                            height: MediaQuery.of(context).size.height,
-                            decoration: BoxDecoration(
-                              color: Color(0xff1e4d2b),
+                          Positioned.fill(
+                            child: ClipShadow(
+                              boxShadow: [
+                                BoxShadow(
+                                  offset: Offset(-10.0, -200.0),
+                                  blurRadius: 0.0,
+                                  spreadRadius: 0.0,
+                                  color: Color(0x4f1e4d2b),
+                                )
+                              ],
+                              clipper: rclip(),
+                              child: Container(
+                                width: MediaQuery.of(context).size.width,
```

Comment:
Redundant code, try using MediaQuery.of(context) syntax only once. We might soon need to create a helper for this.